docs/FOLDER_STRUCTURE.md

```diff
@@ -0,0 +1,69 @@
+
+**It is very important to note that any file ending in an asterisk can be generated from running certain functions in the triangulation and calibration modules. Files and folders not ending in an asterisk must exist before running any functions from sleap-anipose, and must exist in the structure detailed below.** 
+
+```
+Project Root Directory
+├── 02-24-2022/ (Date Directories)
+|        .
+|        .
+|        .
+└── 04-18-2022/
+    ├── 04182022173246/ (Session Directories)
+    |   04182022174515/
+    |        .
+    |        .
+    |        .
+    └── 04182022182439/
+        ├── calibration.toml*
+        |   calibration_metadata.h5* 
+        |   points3d.h5*
+        |   reprojection_errors.png*
+        |   back/ (View Directories)
+        |   mid/
+        |   side/
+        └── top/
+            ├── top-04182022182439-0000_h265_CRF14_denoised.mp4
+            |   top-04182022182439.predictions.slp
+            |   top-04182022182439.predictions.proofread.slp
+            |   reprojected_points2d.h5*
+            |   board_reprojection-0.png*
+            |   board_reprojection-19.png*
+            |   board_reprojection-77.png*
+            |   board_reprojection-124.png*   
+            └── calibration_images/
+                ├── 04182022182439-top-calibration.MOV*
+                |   top-04182022182439-0.jpg
+                |   top-04182022182439-1.jpg
+                |           .
+                |           .
+                |           .
+                └── top-04182022182439-137.jpg
+```
+
+The above structure corresponds to the benchmark dataset and may not be exactly one to one with the way your data was recorded. Even if there are conflicts, there are ways to approximate the benchmark structure so that sleap-anipose can run smoothly. The key aspect of the folder structure is the notion of a session. A session is a folder that contains videos for a singular continuous recording. **A session will only have 1 video per view.** 
+
+
+
+
+Certain files and folders in the example shown above follow a certain naming pattern found all throughout the benchmark dataset. Some naming conventions are detailed below.
+
+- Session directories in the benchmark dataset are named for the date they are recorded on and then the time within the day the recording started.
+
+    For example, session 04182022182439 can be deduced as having started on 04-18-2022 at 18 hours, 24 minutes, 39 seconds. 
+
+- View directories are named are the camera views they represent. 
+
+- The video found in each view folder is named following the convention: `{view}-{session}-{suffix}.mp4`. The '0000_h265_CRF14_denoised' suffix found in the benchmark dataset is a result of the video rendering software.
+
+- The slp files are named following the convention: `{view}-{session}.predictions.{prediction_suffix}.slp`. If the file contains the proofread tracks, the .proofread suffix is added to the file name. 
```

Comment:
do we need to add some mention of the analysis.h5 files?

Reply:
yes, i am re-writing the folder structure file, will update in new pull request